pyctuator/httptrace/http_tracer.py

```diff
@@ -11,4 +12,11 @@ def get_httptrace(self) -> Traces:
         return Traces(list(self.traces_list))
 
     def add_record(self, record: TraceRecord) -> None:
+        headers = record.request.headers
```

Comment:
Hey @mxab , this is very cool thanks.
I'm thinking (out loud) if we want to scrub response headers as well.

Reply:
Hadn't had this use case yet, but I guess it makes sense

Reply:
@michaelyaakoby I fix the scrubber test and added support for the response headers aswell

Reply:
Thanks, this is really awesome. 
Will merge and release ASAP.

Reply:
Cool